frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -239,6 +243,10 @@ export class DeckGlJsonChart extends PureComponent<PropsWithHeight, State> {
             }
             mapboxApiAccessToken={this.props.mapboxToken}
           />
+          <StyledNavigationControlDiv>
```

Comment:
Nit: we usually just describe the element and not specify it specificially. Can just say something like `StyledNavigationControlContainer`